src/styles/normalizes.scss

```diff
@@ -0,0 +1,24 @@
+body {
+  margin: 0;
+  scroll-behavior: smooth;
+  font-family: "Space Grotesk", sans-serif;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  display: flex;
+}
```

Comment:
Setting all `a` tags to `display: flex` is a bad idea because it's an `inline` element and `flex` doesn't work there as expected.